src/backend/t/1000-bs_worker.t

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env perl
+
+use strict;
+use warnings;
+
+use Test::More tests => 3;
+
+
+use FindBin;
+use lib "$FindBin::Bin/lib/";
+
+use Test::Mock::BSConfig;
+use Test::Mock::BSRPC;
+
+
+@::ARGV = ('--testcase');
+require_ok('./bs_worker');
+
+$BSConfig::bsdir = "$FindBin::Bin/data/1000";
```

Comment:
Mybe add
```
my $tmpdir = " $FindBin::Bin/tmp/1000";
BSUtil::rm_rf($tmpdir);
```

Reply:
I modified `$BSConfig::bsdir` to be `$FindBin::Bin`. Otherwise we have problems with extracting the files in other subdirectories.

Now a `$tmpdir` is used, created at the beginning of the tests, and removed afterwards.

Keeping a subdirectory with git wouldn't work. It will be needed an empty `.keep` file to store it in git, and it would be recognized as a spurious file.